libnss_ato.c

```diff
@@ -27,6 +27,26 @@
 #include <string.h>
 #include <stdio.h>
 
+/*
+ * Match to the list of process names for which this module should return a
+ * result.
+ * They are processes which call nss_ato on login, and user-run OS command line
+ * programs.
+ */
+const char *VALID_PROC_NAMES[] = {"sshd",
+                                  "login",
+                                  "w"};
```

Comment:
The short list was taken direct from the ABU version, which has been around for a couple of years, so obvious bugs should have been found by now.
It could be better in terms of flexibility to pull it from config files, but i'd say that is something for future enhancement. given that this is just for authentication via radius, we dont really want other processes taking user mappings, i think.